packages/block-library/src/post-comment-reply-link/index.php

```diff
@@ -0,0 +1,72 @@
+<?php
+/**
+ * Server-side rendering of the `core/post-comment-reply-link` block.
+ *
+ * @package WordPress
+ */
+
+/**
+ * Renders the `core/post-comment-reply-link` block on the server.
+ *
+ * @param array    $attributes Block attributes.
+ * @param string   $content    Block default content.
+ * @param WP_Block $block      Block instance.
+ * @return string Return the post comment's reply link.
+ */
+function render_block_core_post_comment_reply_link( $attributes, $content, $block ) {
+	if ( ! isset( $block->context['commentId'] ) ) {
+		return '';
+	}
+
+	$thread_comments = get_option( 'thread_comments' );
+	if ( ! $thread_comments ) {
+		return '';
+	}
+
+	$comment = get_comment( $block->context['commentId'] );
```

Comment:
When the comment id is invalid, then `$comment` is not an object so it will throw a notice in line 30:

<img width="1766" alt="Screen Shot 2021-10-26 at 17 08 17" src="https://user-images.githubusercontent.com/699132/138907680-69bcc632-3c7b-44ea-81d7-130b7d2dd5ab.png">

We can account for that similar to other blocks with:
```suggestion
	$comment = get_comment( $block->context['commentId'] );
	if ( empty( $comment ) {
		return '';
	}
```